src/actions.tex

```diff
@@ -680,78 +676,70 @@
 
 \actionHeading{Trade \normal{(1\diplopower) (p.~15)}}
 \begin{itemize}
-	\item Must have an available Merchant
+	\item Must have an available (upright) Merchant
 \end{itemize}
 \begin{enumerate}
 	\item Reveal 3 Trade cards
 	\item May collect 2\ducats and end Action
 	\item May move 1 Light \ship
 	\begin{itemize}
-		\item Only if
+		\item Only if selecting a Trade Node where
 		\begin{itemize}
-			\item It increases Trade Power in Node you will select, \conj{or}
-			\item It creates a connection to the Node you will select
+			\item It increases your Trade Power, \conj{or}
+			\item It creates a connection to the Node
 		\end{itemize}
 		\item May displace Opp.'s \ship if no vac. slot
 		\item May not enter a Hostile Sea Zone
 	\end{itemize}
-	\item Select eligible Trade Node (connected to Capital Area by \towns, \vassals, \influence or Light \ships)
+	\item Select an eligible Trade Node
 	\item Select an available (upright) Merchant
 	\begin{itemize}
 		\item Move it to the selected Node
+		\begin{itemize}
+			\item Max 1 Merchant per PR (p.~34)
+		\end{itemize}
 		\item Activate it (lay it on its side)
 	\end{itemize}
-	\item Calculate Tr. Power for all eligible PRs with Merch. in selec. Node and collect \ducats
+	\item Eligible PRs collect the Trade Income\zsavepos{arrowTradeStart}
 \end{enumerate}
-\begin{dynamiccontents*}{frameTrade}\begin{eubox}{\fhTrade}
-	\begin{multicols}{2}
-		\subsubsection*{Merchants \normal{(p.~34-35)}}
-		\begin{itemize}
-			\item Merchant may be activated once per Round
-			\item Trade Node may contain any number of Merchants, but max 1 per player
-		\end{itemize}
 
-		\subsubsection*{Trade Power (\tradepower) \normal{(p.~35)}}
+\begin{dynamiccontents*}{frameTradeIncome}\begin{eubox}{\fhTrade}
+	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}
```

Comment:
Made minor wording adjustments and managed to squeeze some more details into this box. Here's the whole contents of the box after changes:

```
	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}
	\begin{itemize}
		\item Human PRs collect Trade Inc. if the Node
		\begin{itemize}
			\item Has their Merchant, \conj{and}
			\item Is \strong{eligible}: connected to Capital Area by \towns, \vassals, \influence or Light \ships (not required in Inland Trade Node's Area; Occupied \towns/\vassals count)
		\end{itemize}
		\item Expanded - red column, otherwise - green
		\item \strong{1 PR per row} may collect Trade Income. Priority (losers are pushed to lower row):
		\begin{enumerate}
			\item Highest Trade Power
			\item Active PR breaks ties as they please
		\end{enumerate}
		\item Each Pirate Ship lowers all PRs by 1 row
	\end{itemize}
	\smallheading{Secondary Trade Nodes}
	\begin{itemize}
		\item If Active PR has highest (may be tied) Trade Power in eligible Sec. Trade Node
		\begin{itemize}
			\item They may collect Income from the lowest row of the Secondary Node
			\item Other players collect income from Primary Node as normal
		\end{itemize}
		\item May not select a Sec. Node with a Pirate
	\end{itemize}
	\smallheading{Trade Power (\tradepower)}
	\begin{itemize}
		\item 1 \tradepower for your Merchant
		\item 1 \tradepower per Key Province you Own
		\item 1 \tradepower per \ship in Adj. Trade Protection slots (only for Maritime Trade Nodes)
		\item Key Prov. with "+" provide +1\tradepower per "+"
		\item Key Prov. do not count for Sec. Node
	\end{itemize}
	{\botrules
	\subsubsection*{Bot Trade \normal{(p.~4)}}
	\begin{itemize}
		\item Bots have 1 Merchant (never exhausted)
		\item When human PR \action{Trades}, Bots collect Trade Income if Node is eligible for them
		\begin{itemize}
			\item Maritime Trade Node (-1\botpower per Pirate)
			\begin{itemize}
				\item Main Map Nodes: 2\botpower
				\item Distant Nodes: 3\botpower
			\end{itemize}
			\item Inland Trade Node: 2\botpower
			\begin{itemize}
				\item Bot must have 1+ Adjacent \town
			\end{itemize}
			\item -1\botpower if hum. PR has 3+ \tradepower from Key Prov. and \idea{Mercantilism} Idea
		\end{itemize}
		\item When Bot collects Trade Income, move its Merchant to that Node	
	\end{itemize}
	}
```

Reply:
Very nice!